test/test.js

```diff
@@ -95,6 +95,58 @@ describe('exiv2', function(){
       });
     });
   });
+  
+  describe('.setUtf8ImageTags()', function(){
```

Comment:
Yes, I think just adding these two lines to the existing case should be enough to prompt the error..
111. "Xmp.dc.rights": "lang=\"x-default\" © Åge Æble"
121. tags.should.have.property('Xmp.dc.rights', "lang=\"x-default\" © Åge Æble");